contrib/grpcplugins/action/plugin-artifactory-release-bundle-distribute/main.go

```diff
@@ -98,11 +99,57 @@ func (actPlugin *artifactoryReleaseBundleDistributePlugin) Run(ctx context.Conte
 		return actionplugin.Fail("unable to distribute version: %v", err)
 	}
 
+	runResult, err := grpcplugins.GetRunResults(actPlugin.HTTPPort)
+	if err != nil {
+		return actionplugin.Fail("unable to list run results: %v", err)
+	}
+
+	fmt.Printf("Found %d run results\n", len(runResult))
+
+	for _, r := range runResult {
```

Comment:
Is it possible to create a map of repo like in the release plugin to avoid reindex multiple time the same repository ?

Reply:
Done